elastic/apm-server
# Add transaction.experience.longtask metrics fields

## Motivation/summary

Update the schema and fields for RUM to report metrics about longtasks per transaction.

## Checklist


I have considered changes for:
~- [ ] documentation~
~- [ ] logging (add log lines, choose appropriate log selector, etc.)~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
~- [ ] telemetry~
~- [ ] Elasticsearch Service (https://cloud.elastic.co)~
~- [ ] Elastic Cloud Enterprise (https://www.elastic.co/products/ece)~
~- [ ] Elastic Cloud on Kubernetes (https://www.elastic.co/elastic-cloud-kubernetes)~

## How to test these changes

Once https://github.com/elastic/apm-agent-rum-js/issues/895 lands, instrument a page that has long tasks with RUM. Check that the page-load metrics are recorded on the page-load transaction in Elasticsearch.

## Related issues

Closes https://github.com/elastic/apm-server/issues/4226
